src/loaders/express.ts

```diff
@@ -1,10 +1,12 @@
 import express, {Response, Request, NextFunction} from 'express';
 import {RESPONSE_MESSAGES} from '../config/messages';
 import usersRouter from '../routers/users-routes';
+import groupsRouter from '../routers/groups-routes';
 
 export default ({app}: {app: express.Application}) => {
   app.use(express.json());
   app.use('/users', usersRouter);
+  app.use('/groups', groupsRouter);
 
   app.use((req: Request, res: Response, next: NextFunction) => {
```

Comment:
there is a bug: you send response several times. Take a closer look at this middleware